pyproject.toml

```diff
@@ -22,12 +21,13 @@ classifiers=[
 ]
 
 dependencies=[
-    'cpg-flow>=v0.1.2',
+    'cpg-flow>=v0.2.1',
```

Comment:
You should be good to jump up to v0.3.2. That will have massive speed benefits when generating workflow prefixes (hashes), and will come with the `loguru` library for easy logging